docs/index.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to the Guidelines to Join FIWARE Lab
+
+This documentation describes the process to set-up a FIWARE Lab Node
+and to connect it to the FIWARE Lab.
+
+## Table of Contents
+
+1. [Introduction](introduction.md)
+1. [Setting-up a node](node-set-up)
+    1. [Requirements](node-set-up/requirements.md)
+    1. [Installation](node-set-up/installation.md)
+1. [Connecting a node](node-connection)
+    1. [Connecting your Node to FIWARE Lab](node-connection/fiware-lab.md)
+    1. [Registering your Node in FI-Health](node-connection/fi-health.md)
```

Comment:
consistency: Connecting vs Registering? If there's a difference, why there's no Disconnecting below?

Reply:
Connecting are the different steps to connect the services to the central keystone. Registering is more related to the operations to join the Sanity Checks and Monitoring tools of the FIWARE Lab.